contracts/tests/SpigotedLine.t.sol

```diff
@@ -340,8 +340,9 @@ contract SpigotedLineTest is Test, Events {
     }
 
     function test_decrease_unused_revenue(uint buyAmount, uint sellAmount) public {
```

Comment:
@kibagateaux please check that the logic is correct for this, when assesing the difference in `oldClaimTokens` vs `newClaimTokens` in `claimAndTrade()`